postcss.config.js

```diff
@@ -34,7 +34,11 @@ module.exports = ({ file, options, env }) => {
   return {
     plugins: [
       require("postcss-bidirection"),
-      require("autoprefixer"),
+      require("autoprefixer")({
+				browsers: ['last 2 Firefox versions', 'last 2 Chrome versions'],
```

Comment:
should this be "browserslist: [ ..." rather than "browsers: [ ..."?

Reply:
it should be `browsers`

https://github.com/postcss/autoprefixer#options

https://github.com/postcss/autoprefixer/blob/83de1e82c0d418aac4a5fa0faebf71a354babdc3/lib/autoprefixer.js#L56

Reply:
cool nice work!